tests/testthat/test-plot.ppi.R

```diff
@@ -2,29 +2,31 @@ data(example_scan)
 ppi <- project_as_ppi(example_scan)
 example.plot <-plot(ppi, param = "VRADH", zlim = c(-5, 5))
 
-test_that("plot.ppi() returns error on incorrect parameters", {
```

Comment:
In the test title I would retain `plot.ppi()` cf. how you have titled other tests. Affects all tests in this file.